### PR TITLE
Fix prometheus-operator cannot list resource probes error

### PR DESCRIPTION
**Pull request by daniel-hutao**
Signed-off-by: Daniel Hu <farmer.hutao@outlook.com>

Fixes kubesphere/kubesphere/issues/3759

### CODE REVIEW

**Comment by daniel-hutao** [comment]
image

File: roles/ks-monitor/files/prometheus/init/prometheus-operator-clusterRole.yaml [hunk 1]
```diff
@@ -32,6 +32,7 @@ rules:
   resources:
   - alertmanagers
   - alertmanagers/finalizers
+  - probes
```
Comment (benjaminhuo) [inline roles/ks-monitor/files/prometheus/init/prometheus-operator-clusterRole.yaml:35]: Thanks for the PR,
Pls also add probes as below:

```shell
- apiGroups:
  - apiextensions.k8s.io
  resourceNames:
...
  - probes.monitoring.coreos.com
```

**Review by benjaminhuo — CHANGES_REQUESTED** [review]
See my comments above

**Comment by daniel-hutao** [comment]
/need-to-verify

I'll check it again by deploy the latest ks version tonight.

File: roles/ks-monitor/files/prometheus/init/prometheus-operator-clusterRole.yaml (hunk 1 above)
Comment (daniel-hutao) [reply roles/ks-monitor/files/prometheus/init/prometheus-operator-clusterRole.yaml:35]: Ok

**Comment by benjaminhuo** [comment]
You can `git commit -s --amend` and then `git push -f`  to attach modification to the original commit , then run `git fetch upstream` and `git rebase upstream/master` to rebase the upstream changes to make the PR commits clean

**Review by benjaminhuo — APPROVED** [review]

**Comment by daniel-hutao** [comment]
> You can `git commit -s --amend` and then `git push -f` to attach modification to the original commit, then run `git fetch upstream` and `git rebase upstream/master` to rebase the upstream changes to make the PR commits clean in the future @daniel-hutao

Thanks for reminding, I just adjusted the pr and it's suitable now.

**Comment by benjaminhuo** [comment]
> > You can `git commit -s --amend` and then `git push -f` to attach modification to the original commit, then run `git fetch upstream` and `git rebase upstream/master` to rebase the upstream changes to make the PR commits clean in the future @daniel-hutao
> 
> Thanks for reminding, I just adjusted the pr and it's suitable now.

Great!

**Pull request merged**